test/e2e/flowaggregator_test.go

```diff
@@ -427,6 +429,18 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// ToExternalFlows tests the case, where Pod is deployed on the source Node and
+	// sending traffic to an external IP, the flow information is exported as IPFIX
+	// flow records.
+	t.Run("ToExternalFlows", func(t *testing.T) {
+		clientName, clientIPs, cleanupFunc := createAndWaitForPod(t, data, data.createNginxPodOnNode, "test-client-", controlPlaneNodeName())
```

Comment:
why an nginx Pod if we just want to run `curl`?

Reply:
changed to use busybox with `wget`. `curl` is not installed on busybox.
previously I didn't think about the differences between those pod images.
Is that because nginx is used for more complex web services, and busybox is much lightweight?

Reply:
I feel like using an nginx Pod when you don't actually need a Server is just confusing to the reader.